CONTACTS.yaml

```diff
@@ -1,11 +1,55 @@
+#############################################
+# CF Council members:
+# - gh-id: github id
+#   term-start: most recent year of election
 cf-council:
-  - TBD / ...
+  - gh-id: cealsair
+    term-start: 2023
+    role: treasurer
+  - gh-id: ebullient
+    term-start: 2023
+    role: chair
+  - gh-id: mlittle
+    term-start: 2023
+  - ...
+  - ...
+#############################################
+# EGC Project Representatives
+# - gh-org: GitHub project organization
+#   gitlab: URL to gitlab organization
+#   ... (bitbucket, ...)
+#   gh-id: representative's github id
```

Comment:
Does the id need to change for different project orgs or it can be the same?

For instance, would a `gitlab` org mean the id is called `gitlab-id`? Or can `gh-id` be `id` and the org indicates where the id lives?

Reply:
They need a GitHub id.

Reply:
Do we need non GitHub project org urls?

If a project is entirely on Gitlab, we require them to set up a GitHub account if they don't already have one?

Reply:
For generation of links on the website, yes. Name of the project, the root of information about it (could be the github org, could be another code-repository. We could decide to just have the url for the project, and let it be whatever the project wants).

Important part: Project name, some representation of its scope (i.e. Hibernate GitHub org includes several subprojects.. they're all together), and the GH id of the EGC representative for it.